main.go

```diff
@@ -39,6 +38,13 @@ var (
 	errParseECPoint  = "failed to parse EC point"
 )
 
+const (
+	// January 1, 2022
```

Comment:
Documenting `date -u +%s -d 2022-01-01T00:00:00Z` * 1000 might be helpful for later updates.